src/uniswap-summaries.md

```diff
@@ -3535,6 +3535,56 @@ module SOLIDITY-UNISWAP-APPROVE-SUMMARY
 endmodule
 ```
 
+```k
+module SOLIDITY-UNISWAP-TRANSFERFROM-SUMMARY
+  imports SOLIDITY-CONFIGURATION
+  imports SOLIDITY-EXPRESSION
+  imports SOLIDITY-UNISWAP-TOKENS
+
+  rule <k> transferFrom:Id ( v(V1:MInt{160}, address #as T), v(V2:MInt{160}, T), v(V3:MInt{256}, uint256), .TypedVals ) ~> Ss => v ( true , bool ) ~> Ss </k>
+        <summarize> true </summarize>
+        <this> THIS </this>
+        <contract-address> THIS </contract-address>
```

Comment:
You can match `<this-type>` and `<contract-id>` cells. I posted details on slack.